src/style/index.less

```diff
@@ -79,6 +79,7 @@ pre.highlight {
 
 content-region {
 	display: block;
+	min-height: calc(~"100vh - 256px"); // hack for less compiler which actually subtracts px from vh
```

Comment:
this actually relies on CSS calc, which has somewhat poor browser support.  I wonder - would #56 fix this more simply?  Cheers!